config/settings.yml

```diff
@@ -1082,7 +1082,7 @@
   :name: EVM
   :rails_server: puma
   :remote_console_type: VMRC
-  :role: database_operations,event,reporting,scheduler,smartstate,ems_operations,ems_inventory,user_interface,websocket,web_services,automate
+  :role: database_operations,event,reporting,scheduler,smartstate,ems_operations,ems_inventory,user_interface,websocket,cockpit_ws,web_services,automate
```

Comment:
I believe we shouldn't have this role enabled by default.  I'm not sure if this was enabled for testing purposes or you want this to run by default.